src/org/zaproxy/zap/extension/httpsinfo/ExtensionHttpsInfo.java

```diff
@@ -15,45 +15,49 @@
  * See the License for the specific language governing permissions and 
  * limitations under the License. 
  */
-package org.zaproxy.zap.extension.httpsInfo;
+package org.zaproxy.zap.extension.httpsinfo;
 
 import java.awt.EventQueue;
 import java.awt.event.WindowAdapter;
 import java.awt.event.WindowEvent;
+import java.io.IOException;
 import java.net.MalformedURLException;
 import java.net.URL;
 import java.util.ArrayList;
 import java.util.List;
 
+import javax.swing.ImageIcon;
+
+import org.apache.log4j.Logger;
 import org.parosproxy.paros.Constant;
 import org.parosproxy.paros.extension.ExtensionAdaptor;
 import org.parosproxy.paros.extension.ExtensionHook;
+import org.parosproxy.paros.network.HttpMessage;
 
 /*
  * An example ZAP extension which adds a right click menu item to all of the main
  * tabs which list messages. 
  * 
  * This class is defines the extension.
  */
-public class RightClickMenu extends ExtensionAdaptor {
+public class ExtensionHttpsInfo extends ExtensionAdaptor {
+
+	public static final String NAME = "ExtensionHttpsInfo";
+	public static final String ICON_PATH = "/org/zaproxy/zap/extension/httpsinfo/resources/icon.png";
+	private static final Logger LOGGER = Logger.getLogger(ExtensionHttpsInfo.class);
 
 	private MenuEntry httpsEntry = null;
 	private List<MDialog> dialogues;
 	private boolean unloaded;
 
-	public RightClickMenu() {
+	public ExtensionHttpsInfo() {
 		super();
-		initialize();
 	}
 
-	public RightClickMenu(String name) {
+	public ExtensionHttpsInfo(String name) {
```

Comment:
This constructor can be removed (it's not required by core code and it's probably not going to be used by 3rd-party code either).